### PR TITLE
W1150n/adding laptop security guide

### PR DESCRIPTION
**Pull request by w1150n**

### CODE REVIEW

File: process/securing_your_workspace.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+With a few simple tricks you can secure your machine.
+
+* Enable FileVault. Your hard drive needs to be encrypted to protect the data should the machine be stolen.
+* Require a password to login. Fresh reboots should not automatically log you in.
+* Enable Auto-lock/sleep after 10 minutes.
+* Never leave your laptop unlocked. Don't walk away for lunch or an errand without locking your computer.
+* Keep your laptop in a safe place. e.g. Not in your car. Cars are broken into very often.
+
+And finally, if your workspace is compromised physically or digitally, please notify Chris, Matt and Dean immediately.
```
Comment (holetse) [inline process/securing_your_workspace.md]: I would add the your team lead to that list.

**Comment by holetse** [comment]
Maybe a comment about using 1password or LastPass. Browser password managers store the password in basically clear text. Though FileVault solves that.

File: process/securing_your_workspace.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Securing Your Workspace
+
+With a few simple tricks you can secure your machine.
+
+* Enable FileVault. Your hard drive needs to be encrypted to protect the data should the machine be stolen.
+* Require a password to login. Fresh reboots should not automatically log you in.
+* Enable Auto-lock/sleep after 10 minutes.
+* Never leave your laptop unlocked. Don't walk away for lunch or an errand without locking your computer.
+* Keep your laptop in a safe place. e.g. Not in your car. Cars are broken into very often.
```
Comment (JeffreyATW) [inline process/securing_your_workspace.md]: Maybe clarify that trunk is better than plain-sight?

File: process/securing_your_workspace.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Securing Your Workspace
+
+With a few simple tricks you can secure your machine.
+
+* Enable FileVault. Your hard drive needs to be encrypted to protect the data should the machine be stolen.
+* Require a password to login. Fresh reboots should not automatically log you in.
+* Enable Auto-lock/sleep after 10 minutes.
+* Never leave your laptop unlocked. Don't walk away for lunch or an errand without locking your computer.
+* Keep your laptop in a safe place. e.g. Not in your car's front seat. Cars are broken into very often. If you must leave it in your car, put it in the trunk or hide it well.
+* Use a password manager, like 1Password to keep your passwords hard/secure between sites.
```
Comment (holetse) [inline process/securing_your_workspace.md]: I'd add LastPass to this list too. As it's free and pretty solid.

File: process/securing_your_workspace.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Securing Your Workspace
```
Comment (greacen) [inline process/securing_your_workspace.md]: Other items worth noting in this doc: 
 * Physical security for the office: what's expected?
 * How about 'after the project is over'. Any special concerns there?  
 * How to escalate when something goes wrong.

**Pull request merged**